README.rst

```diff
@@ -1,73 +1,14 @@
 ETFL
 ====
-|Documentation Status| |Build Status| |Codecov| |Codacy branch grade| |license| |doi|
```

Comment:
@oftadehomid your commit removes important documentation from the readme. Would you please fix this ? 
Could you also check that it does not break the documentation elsewhere?
Cheers,
Pierre